File: seungyoon/22_12_02_02.py
def solution(num_list, n):
    arr = [[0 for j in range(n)] for i in range(len(num_list) // n)]
    
    q = 0
    
    for t in range(len(num_list) // n):
        for y in range(n):
            if q < len(num_list):
                arr[t][y] = num_list[q]
                q += 1

        
    return arr

File: seungyeop/22_12_02_02.java
class Solution {
    public int[][] solution(int[] num_list, int n) {
        int row = num_list.length / n;
        int[][] answer = new int[row][n];
        for (int i = 0; i < row; i++) {
            for (int j = 0; j < n; j++) {
                answer[i][j] = num_list[i*n+j];
            }
        }
        return answer;
    }
}
